### DISCUSSION TITLE
我不建议在程序报错时弹出的窗口的确认按钮附加“复制到剪切板”功能

### DISCUSSION THREAD

**Discussion by FTS427** [MCSLTeam/MCSL2#110] [功能建议] [answered]
如下图，我不建议把“复制到剪切板”功能和“确认”按钮绑定在一起，我觉得将报错信息复制到剪切板是可选的，可以独立出来，我不希望在我已经知道了这个问题后按下“确认”程序会把我当前剪切板里的内容换掉
![image](https://github.com/MCSLTeam/MCSL2/assets/122330825/6474fd17-6f11-490b-a804-a2406521d0ed)

**Comment by lxhtt** [comment, marked as answer]
有取消按钮。

**Comment by FTS427** [comment]
好吧
